src/components/charts/discoverChartDetail/DiscoverChartDetails.tsx

```diff
@@ -212,7 +212,7 @@ const DiscoverChartDetails: React.FC<DiscoverChartDetailsProps> = ({ match, hist
             </div>
             <Switch>
                 <Route
-                    path={`${URLS.CHARTS}/discover/chart/:chartId/deploy-chart`}
+                    path={`${URLS.CHARTS}/discover/chart/:chartId/deploy-chart/:presetValueId?`}
```

Comment:
Please use constant values here for values like `discover` , `chart` from URLS

Reply:
Done